packages/dmn-js-shared/src/features/modeling/behavior/IdChangeBehavior.js

```diff
@@ -47,7 +47,8 @@ export default class IdChangeBehavior extends CommandInterceptor {
   }
 
   shouldSkipUpdate(bo, oldProperties, newProperties) {
-    return !isIdChange(oldProperties, newProperties) ||
+    return !oldProperties ||

```

Comment:
I like that change. Does it have any negative side-effects?

Reply:
I wouldn't assume it, quick testing inside the Modeler didn't throw any error, also the tests are good.

Reply:
Also, skipping the update when there is nothing to update makes absolute sense.